pkg/params/triggertype/types.go

```diff
@@ -48,7 +50,7 @@ const (
 	CheckSuiteRerequested Trigger = "check-suite-rerequested"
 	Comment               Trigger = "comment"
 	Incoming              Trigger = "incoming"
-	LabelUpdate           Trigger = "label_update"
+	LabelUpdate           Trigger = "pull_request_labeled"
```

Comment:
```suggestion
	PullRequestLabeled           Trigger = "pull_request_labeled"
```

shouldn't the var name be changed

Reply:
Ah, makes sense. I think I can't directly commit this since I believe this variable might be used in other packages as well. Will make changes accordingly.